src/components/Content/TaskHierarchy/TaskHierarchy.js

```diff
@@ -15,7 +15,7 @@ class TaskHierarchy extends Component {
   }
 
   componentWillMount() {
-    const pursuanceID = this.props.match.params.pursuance_id;
+    const pursuanceID = this.props.pursuanceId;
```

Comment:
@Spetastian You're right, let's stick with the convention you used here and call the variable `pursuanceId` rather than `pursuanceID` :+1:

Reply:
Old habits die hard. I really dont like having names with underscore or uppercased words in JavaScript :smile: 

... unleass its a constant of sorts...